smithy-model/src/test/resources/software/amazon/smithy/model/loader/valid/main-test.smithy

```diff
@@ -28,79 +32,93 @@ operation GetCurrentTime {
 }
 
 structure GetCurrentTimeOutput {
-  time: smithy.api#Timestamp
+    time: smithy.api#Timestamp
 }
 
 @readonly
 operation GetForecast {
-    input: GetForecastInput,
-    output: GetForecastOutput,
+    input: GetForecastInput
+    output: GetForecastOutput
     errors: [NoSuchResource]
 }
 
 structure GetForecastInput {
-  @required cityId: CityId
+    @required
+    cityId: CityId
 }
 
 structure GetForecastOutput {
-  @required chanceOfRain: smithy.api#Float,
-  @required low: smithy.api#Integer,
-  @required high: smithy.api#Integer
+    @required
+    chanceOfRain: smithy.api#Float
+    @required
+    low: smithy.api#Integer
+    @required
+    high: smithy.api#Integer
 }
 
 @readonly
 operation GetCity {
-    input: GetCityInput,
-    output: GetCityOutput,
+    input: GetCityInput
+    output: GetCityOutput
     errors: [NoSuchResource]
 }
 
 structure GetCityInput {
-  @required cityId: CityId
+    @required
+    cityId: CityId
 }
 
 structure GetCityOutput {
-  @required name: smithy.api#String,
-  @required coordinates: CityCoordinates
+    @required
+    name: smithy.api#String
+    @required
+    coordinates: CityCoordinates
 }
 
 @readonly
-@paginated(inputToken: "nextToken", outputToken: "nextToken",
-           pageSize: "pageSize", items: "items")
+@paginated(inputToken: "nextToken", outputToken: "nextToken", pageSize: "pageSize", items: "items")
```

Comment:
This change is interesting. How do you decide when to use a single line vs multiple lines for a trait? Lots of traits tend to use a single line, so I like this (like http traits), but some like this one can have lots of properties.